build.gradle

```diff
@@ -38,7 +38,7 @@ ext {
     buildToolsVersion = '28.0.3'
 
     // App dependencies
-    supportLibraryVersion = '1.1.0'
+    supportLibraryVersion = '1.0.0'
```

Comment:
what does it do exactly?

Reply:
The recycler view in Account Section is damaged.